tasks/main.yml

```diff
@@ -389,3 +389,11 @@
   yum:
     name: "{{ item.name }}"
   with_items: "{{ yum_packages }}"
+
+- name: Drone client
+  tags:
+    - drone
+  unarchive:
+    src: http://downloads.drone.io/release/linux/amd64/drone.tar.gz
```

Comment:
This shouldn't be hardcoded but using varilables. Both URL and destination should be set in defaults/main.yml. As an example see L239 `mc | Install latest binary release.`